packages/grafana-ui/src/components/uPlot/config/UPlotScaleBuilder.ts

```diff
@@ -94,6 +95,10 @@ export class UPlotScaleBuilder extends PlotConfigBuilder<ScaleProps, Scale> {
       return minMax;
     };
 
+    if (isBooleanUnit(scaleKey)) {
+      console.log('only set ticks for true/false?  force range 0/1?');
```

Comment:
@leeoniya -- any thoughts on how to get a boolean axis to only show ticks at 1/0?  not all the points in between?

Reply:
`axis.splits: () => [0, 1]`

Reply:
probably also want `scale.range: () => [0, 1]`